script-base.js

```diff
@@ -74,7 +90,7 @@ Generator.prototype.addChangelogToLiquibase = function (changelogName) {
             file: fullPath,
             needle: '<!-- JHipster will add liquibase changelogs here -->',
             splicable: [
-                    '<include file="classpath:config/liquibase/changelog/' + changelogName + '.xml" relativeToChangelogFile="false"/>'
```

Comment:
this doesn't have anything to do with i18n -> can you remove this? This could be discussed in a separate thread